File: app/src/main/java/com/finacedavid/domain/model/Payable.kt
package com.finacedavid.domain.model

import java.math.BigDecimal
import java.time.LocalDate

data class Payable(
    val id: Long = 0,
    val title: String,
    val amount: BigDecimal,
    val dueDate: LocalDate,
    val status: PayableStatus,
    val note: String?
)

enum class PayableStatus { Pendente, Pago }

File: desktop/FinaceDavid.Desktop/Models/Payable.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FinaceDavid.Desktop.Models;

public enum PayableStatus
{
    Pendente,
    Pago
}

public class Payable
{
    [Key]
    public int Id { get; set; }
    [Required]
    public string Title { get; set; } = string.Empty;
    [Column(TypeName = "decimal(18,2)")]
    public decimal Amount { get; set; }
    public DateTime DueDate { get; set; }
    public PayableStatus Status { get; set; }
    public string Notes { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}
